apollo-router/src/trace.rs

```diff
@@ -62,7 +62,16 @@ pub(crate) fn try_initialize_subscriber(
             let provider = builder.with_span_processor(batch).build();
 
             let tracer = provider.tracer("opentelemetry-jaeger", Some(env!("CARGO_PKG_VERSION")));
-            let _ = opentelemetry::global::set_tracer_provider(provider);
```

Comment:
oh geez I should have seen that xD

Reply:
Yeah, it's a tricky one. It's hard to know what the correct thing to do is in this scenario.